README.md

```diff
@@ -1,20 +1,50 @@
 [![Documentation Status](https://readthedocs.org/projects/deepcell-kiosk-figure-generation/badge/?version=latest)](https://deepcell-kiosk-figure-generation.readthedocs.io/en/latest/?badge=latest)
 
-Recreate the figures from https://www.biorxiv.org/content/10.1101/505032v3 in 4 easy steps!
-
+The DeepCell Kiosk paper [(Bannon et al., 2020)](<https://www.biorxiv.org/content/10.1101/505032v3>) presents cost and runtime benchmarks for running a generic image segmentation pipeline on datasets of given sizes inside the DeepCell Kiosk with different sets of constraints (Fig. 1b).
+ 
 N.B.: Before following any of these steps, I'd recommend activating a Python `virtualenv`. It won't affect how easy it is to run this project, probably, but it will make your life easier after you've forgotten about this project and moved on to other Python packages.
 
 (To view API documentation for this project online, please go [here](https://deepcell-kiosk-figure-generation.readthedocs.io).)
 
 ## Figure Creation
 1) To create the figures, first install the `deepcell-kiosk-figure-generation` package using the included `setup.py` file with the command `pip install -e .`.
```

Comment:
Remove the periods at the end of these lists. `This formatting makes it difficult to distinguish this . from the end of a line`

Reply:
Good call. Done.